analysis/topEFT/topeft.py

```diff
@@ -332,7 +332,7 @@ def process(self, events):
           
             # Trigger SF
             GetTriggerSF(year,events,l0,l1)
-
+            
```

Comment:
Also, this is not very important, but could you remove this added whitespace? Other than just being unnecessary, it might also make the PR slightly less clear if we look back at this in the future (since I think this PR does not need to change the processor at all, but at first glance we might think that it does until we look more closely and see that it was just whitespace that was added).